cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -107,13 +111,19 @@ Conditions:
   UseAssociatePublicIpAddress: !Equals
     - !Ref 'AssociatePublicIpAddress'
     - true
+  CreateEC2IAMRoleCondition: !Equals
+    - !Ref 'CreateEC2IAMRole'
+    - 'true'
```

Comment:
nit: for consistency you can avoid quoting `true`

Reply:
Done